osu.Game/Overlays/Notifications/SimpleNotification.cs

```diff
@@ -36,48 +36,44 @@ public IconUsage Icon
             set
             {
                 icon = value;
-                iconDrawable.Icon = icon;
+                if (iconDrawable != null)
+                    iconDrawable.Icon = icon;
             }
         }
 
-        private readonly TextFlowContainer textDrawable;
-        private readonly SpriteIcon iconDrawable;
+        private TextFlowContainer? textDrawable;
 
-        protected Box IconBackground;
+        private SpriteIcon? iconDrawable;
 
-        public SimpleNotification()
+        [BackgroundDependencyLoader]
+        private void load(OsuColour colours, OverlayColourProvider colourProvider)
         {
+            Light.Colour = colours.Green;
+
             IconContent.AddRange(new Drawable[]
             {
-                IconBackground = new Box
+                new Box
                 {
                     RelativeSizeAxes = Axes.Both,
-                    Colour = ColourInfo.GradientVertical(OsuColour.Gray(0.2f), OsuColour.Gray(0.6f))
+                    Colour = colourProvider.Background5,
                 },
                 iconDrawable = new SpriteIcon
                 {
                     Anchor = Anchor.Centre,
                     Origin = Anchor.Centre,
                     Icon = icon,
-                    Size = new Vector2(20),
+                    Size = new Vector2(16),
                 }
             });
 
-            Content.Add(textDrawable = new OsuTextFlowContainer(t => t.Font = t.Font.With(size: 14))
+            Content.Add(textDrawable = new OsuTextFlowContainer(t => t.Font = t.Font.With(size: 14, weight: FontWeight.Medium))
```

Comment:
`ProgressNotification`'s text flow doesn't have the same weight spec that was added here - is this intentional? Note that it's a completely disparate class, in that we have

```
Notification
   SimpleNotification
   ProgressNotification
```

(Notably, it also doesn't have the same font size either, although that is a pre-existing issue that I didn't even notice until now...)

Reply:
Well spotted.

It's a weird hierarchy. A lot of the shared pieces should probably be simplified into the base `Notification` class. I even considered making a single `Notification` which has optional progress. But it was all a bit too much unnecessary refactor to get this feature completed.

I'll adjust specs to match for now.